src/rust/engine/process_execution/src/local.rs

```diff
@@ -178,6 +178,16 @@ impl HermeticCommand {
   }
 }
 
+// TODO: A Stream that ends with `Exit` is error prone: we should consider creating a Child struct
```

Comment:
Track with a Pants issue?

Reply:
This one is not the source of any trouble right now, afaik. So I don't think it meets the bar for an issue at the moment.